cdc/cdc/metrics.go

```diff
@@ -19,15 +19,9 @@ import (
 	"github.com/tikv/migration/cdc/cdc/kv"
 	"github.com/tikv/migration/cdc/cdc/owner"
 	"github.com/tikv/migration/cdc/cdc/processor"
-	tablepipeline "github.com/tikv/migration/cdc/cdc/processor/pipeline"
+	keyspanpipeline "github.com/tikv/migration/cdc/cdc/processor/pipeline"
```

Comment:
why not just use the original name `pipeline`?

Reply:
I don't know why tablepipeline was used instead of pipeline. 🤣